JavaScript/React.md

```diff
@@ -509,10 +559,33 @@ export class ComponentName extends React.Component {
 }
 ```
 
+#### Functional components
+
+In functional components it's fine to use either `useState` or `useLocalStore`, but don't mix them together - choose one for the project and then stick with it.
+`useLocalStore` might be simpler in the short run, but it might not work with new React features like concurrent rendering.
```

Comment:
Not sure if we should leave this to a developers choice or should we choose.

Reply:
Just add: `we recommend using...`

Reply:
Maybe we can add this link somewhere https://mobx-react.js.org/